tutorials/sample_vqe_program/qiskit_runtime_vqe_program.ipynb

```diff
@@ -0,0 +1,1394 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "a52bdcdf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#provider.runtime.delete_program(program_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b521258",
+   "metadata": {},
+   "source": [
+    "## Running the program\n",
+    "\n",
+    "### Specify parameters\n",
+    "\n",
+    "To run the program we need to specify the `options` that are used in the runtime environemnt (not the program variables).  At present, only the `backend_name` is required."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "10629ac7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "backend = provider.backend.ibmq_qasm_simulator\n",
+    "options = {'backend_name': backend.name()}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "55e52ecb",
+   "metadata": {},
+   "source": [
+    "The `inputs` dictionary is used to pass arguements to the `main` function itself.  For example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "d113b419",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "inputs = {}\n",
+    "inputs['hamiltonian'] = H\n",
+    "inputs['optimizer_config']={'maxiter': 10}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "55831f5a",
+   "metadata": {},
+   "source": [
+    "### Execute the program\n",
+    "\n",
+    "We now can execute the program and grab the result."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "fa79f669",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = provider.runtime.run(program_id, options=options, inputs=inputs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "e2ef686a",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'fun': -0.2177734375,\n",
+       " 'x': array([-0.37738882,  3.66268048, -0.37574463,  5.18112796,  0.26720409,\n",
+       "        -0.96836968,  3.74224556,  4.31084386,  1.63188765,  5.151062  ,\n",
+       "         6.93889342,  4.1269752 ,  2.2251061 ,  5.48401816,  2.24756001,\n",
+       "         5.99239828]),\n",
+       " 'nit': 10,\n",
+       " 'nfev': 20,\n",
+       " 'message': 'Optimization terminated successfully.',\n",
+       " 'success': True}"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "job.result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a75f75cd",
+   "metadata": {},
+   "source": [
+    "A few things need to be pointed out. First, we did not get back any interm results, and second the return object is a plain dictionary.  This is because we did not listen for the return results, and we did not tell the job how to format the return result."
```

Comment:
```suggestion
    "A few things need to be pointed out. First, we did not get back any interim results, and second the return object is a plain dictionary.  This is because we did not listen for the interim results, and we did not tell the job how to format the return result."
```
I think there are several copied&pasted `interm`